lib/temporal/retry_policy.rb

```diff
@@ -0,0 +1,39 @@
+require 'temporal/errors'
+
+module Temporal
+  # See https://docs.temporal.io/application-development/features/#workflow-retry-policy
+  class RetryPolicy < Struct.new(
```

Comment:
Can you set defaults here? We have a pretty common set of defaults.

Reply:
Will do 👍